### DISCUSSION TITLE
pymc-marketing installation question

### DISCUSSION THREAD

**Discussion by sreedta8** [pymc-labs/pymc-marketing#899] [Q&A] [answered]
I would like to install and use pymc-marketing without
1. using conda or mamba based paths (without using Anaconda)
2. without resorting to Docker-based builds

I have Python 3.11 ad I have created virtual envs to run "pymc", "tensorflow/keras", "pytorch"on a cpu-based machine. I also have "jupyter" installed. Can I create a virtual env for "pymc-marketing"using installation via pypi and pip? Could I, instead of creating a separate env for "pymc-marketing", simply install the same into the "pymc" virtual env?

Could anyone point me to the reference document that would help me in this regard?

**Comment by williambdean** [comment, marked as answer]
You can just pip install pymc-marketing then. The recommendation to conda install really comes from pymc rather than pymc marketing. But if you have pymc installed and all of the system dependencies to work to your liking, you should be good to go

**Reply by sreedta8** [reply]
Thanks a lot @wd60622 for the follow-up. I will try and report back
